src/GovUk.Education.ExploreEducationStatistics.Content.Model/Release.cs

```diff
@@ -24,6 +24,8 @@ public class Release : ICreatedUpdatedTimestamps<DateTime, DateTime?>
 
     public required int Year { get; set; }
 
+    public string? Label { get; set; }
```

Comment:
You could consider adding a change to ReleaseGeneratorExtensions to support `Label`. It might help avoid merge conflicts if both EES-5628 and EES-5634 both come along at the same time and need it.

Reply:
Good shout. I'll add it